ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ConsoleApp
+{
+    //simple bubble algorithm
+    class Program
+    {
+        static void Main()
+        {
+            int[] arr = { 800, 11, 50, 771, 649, 770, 240, 9 };
+
+            for (int i = 0; i < arr.Length; i++)
+            {
+                for (int j = 0; j < arr.Length - 1; j++)
```

Comment:
you do not need to go through whole array each time in the inner loop, you can stop on **arr.Length - i** element